Tests/SourceKittenFrameworkTests/Fixtures/Expression.json

```diff
@@ -0,0 +1,110 @@
+{
+  "Expression.swift": {
+    "key.diagnostic_stage": "source.diagnostic.stage.swift.parse",
+    "key.substructure": [
+      {
+        "key.filepath": "Expression.swift",
+        "key.parsed_declaration": "class Hidden",
+        "key.annotated_decl": "<Declaration>class Hidden</Declaration>",
+        "key.fully_annotated_decl": "<decl.class><syntaxtype.keyword>class</syntaxtype.keyword> <decl.name>Hidden</decl.name></decl.class>",
+        "key.accessibility": "source.lang.swift.accessibility.internal",
+        "key.length": 15,
+        "key.runtime_name": "_TtC8__main__6Hidden",
+        "key.parsed_scope.start": 1,
+        "key.typeusr": "_Tt",
+        "key.kind": "source.lang.swift.decl.class",
+        "key.offset": 0,
+        "key.nameoffset": 6,
+        "key.typename": "Hidden.Type",
+        "key.actionable": [
+          {
+            "key.actionname": "Rename"
+          }
+        ],
+        "key.name": "Hidden",
+        "key.usr": "s:C10Expression6Hidden",
+        "key.bodyoffset": 14,
+        "key.bodylength": 0,
+        "key.parsed_scope.end": 1,
+        "key.namelength": 6
+      },
+      {
+        "key.filepath": "Expression.swift",
+        "key.parsed_declaration": "class HiddenUsage",
+        "key.annotated_decl": "<Declaration>class HiddenUsage</Declaration>",
+        "key.fully_annotated_decl": "<decl.class><syntaxtype.keyword>class</syntaxtype.keyword> <decl.name>HiddenUsage</decl.name></decl.class>",
+        "key.accessibility": "source.lang.swift.accessibility.internal",
+        "key.length": 126,
+        "key.substructure": [
+          {
+            "key.filepath": "Expression.swift",
+            "key.parsed_declaration": "func functionWithHiddenDependency()",
+            "key.annotated_decl": "<Declaration>func functionWithHiddenDependency()</Declaration>",
+            "key.fully_annotated_decl": "<decl.function.method.instance><syntaxtype.keyword>func</syntaxtype.keyword> <decl.name>functionWithHiddenDependency</decl.name>()</decl.function.method.instance>",
+            "key.accessibility": "source.lang.swift.accessibility.internal",
+            "key.length": 100,
+            "key.substructure": [
+              {
+                "key.namelength": 6,
+                "key.nameoffset": 100,
+                "key.length": 8,
+                "key.name": "Hidden",
```

Comment:
Is this all the detail we can get about this function (which is really an initializer for the `class Hidden`)? It would be great to have a reference of some kind to the declaration of the initializer. Although in this case, that initializer is generated by the compiler and there is no code to reference...